frontend/src/console/components/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+// import messages from './displayMessages';
+import './styles.scss';
+import { Col, Row } from 'react-bootstrap';
+import { ThemeSchema } from 'ocim-client';
+import avatar from 'assets/avatar-default.png';
+// import { CustomizableButton } from '../CustomizableButton';  # TODO: uncomment in BB-2219.
+import { CustomizableLink } from '../CustomizableLink';
+import { InstanceSettingsModel } from '../../models';
+
+interface NavigationMenuProps {
+  children?: React.ReactNode;
+  instanceData: InstanceSettingsModel | null;
+  themeData: ThemeSchema;
+  loggedIn?: boolean;
+}
+
+export const NavigationMenu: React.FC<NavigationMenuProps> = (
+  props: NavigationMenuProps
+) => {
+  const { themeData } = props;
+
+  const userMenu = props.loggedIn ? (
+    <Row className="navigation-menu navigation-submenu">
+      <Col>
+        <CustomizableLink linkColor={themeData.mainNavLinkColor}>
+          Help
+        </CustomizableLink>
+        <img src={avatar} alt="Avatar" />
+        <CustomizableLink linkColor={themeData.mainNavLinkColor}>
+          JoeSoap
+        </CustomizableLink>
+        <CustomizableLink linkColor={themeData.userDropdownColor}>
+          <i className="fas fa-caret-down" />
+        </CustomizableLink>
+      </Col>
+    </Row>
+  ) : (
+    <Row className="navigation-menu navigation-submenu">
+      <Col>
+        <CustomizableLink>Register</CustomizableLink>
+        <CustomizableLink>Sign in</CustomizableLink>
+        {/* TODO: Dummy buttons. While merging BB-2219 remove these from above and uncomment the ones below. */}
+        {/* <CustomizableButton */}
+        {/*  initialTextColor={themeData.btnRegisterColor} */}
+        {/*  initialBackgroundColor={themeData.btnRegisterBg} */}
+        {/*  initialBorderColor={themeData.btnRegisterBorderColor} */}
+        {/*  initialHoverTextColor={themeData.btnRegisterHoverColor} */}
+        {/*  initialHoverBackgroundColor={themeData.btnRegisterHoverBg} */}
+        {/*  initialHoverBorderColor={themeData.btnRegisterHoverBorderColor} */}
+        {/* > */}
+        {/*  Register */}
+        {/* </CustomizableButton> */}
+        {/* <CustomizableButton */}
+        {/*  initialTextColor={themeData.btnSignInColor} */}
+        {/*  initialBackgroundColor={themeData.btnSignInBg} */}
+        {/*  initialBorderColor={themeData.btnSignInBorderColor} */}
+        {/*  initialHoverTextColor={themeData.btnSignInHoverColor} */}
+        {/*  initialHoverBackgroundColor={themeData.btnSignInHoverBg} */}
+        {/*  initialHoverBorderColor={themeData.btnSignInHoverBorderColor} */}
+        {/* > */}
+        {/*  Sign in */}
+        {/* </CustomizableButton> */}
+      </Col>
+    </Row>
+  );
+
+  return (
+    <Row
+      className="main-navigation-menu navigation-menu"
+      style={{ background: themeData.mainNavColor }}
```

Comment:
The nav bar is showing up as transparent by default for me. The same happens when clicking on the reset button in the background color field.
![image](https://user-images.githubusercontent.com/27893385/77761660-2412ec00-7017-11ea-9a4c-15ddd8fb5656.png)

Can you add the LMS default colors if `themeData` doesn't have those set?

Reply:
Done, I also fixed the "bumping" text on hover.